miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebChromeClient.kt

```diff
@@ -38,7 +45,19 @@ internal class MiniAppWebChromeClient(
         origin: String?,
         callback: GeolocationPermissions.Callback?
     ) {
-        callback?.invoke(origin, true, false)
+        val hasCustomPermission =
+            miniAppCustomPermissionCache.readPermissions(miniAppInfo.id)
+                .pairValues.find {
+                    it.first == MiniAppCustomPermissionType.LOCATION
+                }?.let {
+                    it.second == MiniAppCustomPermissionResult.ALLOWED
+                }
+        val hasDevicePermission = ActivityCompat.checkSelfPermission(
```

Comment:
Do we need to have this check for the device permission? I think this is already handled by the android system. Device permissions was already working correctly without this.

Reply:
Thanks for the concern, we can skip this also. Initially, I thought we should check this but it's not obvious in this situation, so I'll update this part.